docs/docs/parachains/composable/composable-council.md

```diff
@@ -0,0 +1,125 @@
+
+Additional Councils established after the initial Composable Council are able to initiate proposal creation and develop their own proposals. Some examples for proposal topics are as follows:
+
+- Implementation of a Crowdloan Rewards pallet enabling those who contributed to the [Composable Crowdloan](../composable/composable-crowdloan.md) to claim their rewards
+- Creation of new pallets, products, and protocols on the Composable parachain
+
+All council members have veto power, which can be used once per proposal. If a council member vetoes a proposal, they cannot veto the same proposal if it is resubmitted after the 15-day cooldown period.
+
+:::caution Emergency Measures
+In case of extreme urgency, the Council may implement an “emergency measure”, such as when a major security vulnerability is found or when the imminence of great prejudice to the token holders or the network is clear, which does not need to be submitted as a proposal or for approval in a referendum. 
+
+In such a case, the emergency measure will be immediately executable and implemented as fast as the Council deems fit.
+:::
+
+### Technical Committee
+
+The Technical Committee for the Composable parachain consists of core developers who are chosen by the Council. The members of the Technical Committee will initially be composed of Composable Finance developers. The role of the Technical Committee includes, among others, ensuring the technical stability and critical safety measures of the parachain. 
+
+#### Emergency Proposals
+
+The Technical Committee may, along with the Council, submit an “emergency proposal”, which should not be confused with an “emergency measure”. An emergency proposal submitted by the technical committee will be submitted for immediate referendum, along with fast-tracked voting and implementation. This will be the method used for emergency bug fixes or rapid implementation of new features into the Composable network, and in general, the strategic furtherance of the Composable Network’s technology.
+
+## Democracy guidelines
+
+Proposals can be initiated by both the Community and the Council, and are required for any action that would directly affect the Composable parachain such as approving or rejecting a treasury payout, code upgrades, graduating pallets, and more. The following table portrays the initial criterion for interacting with governance proposals on the Composable parachain:
+
+| Parameter                                          | Period/Number  |
+|----------------------------------------------------|----------------|
+| Referendum voting period                           | 5 days          |
+| Enactment delay of an approved referendum          | 2 days         |
+| Cool-off period after a proposal has been rejected | 7 days        |
+| Maximum pending community proposals                | 100 |
+
+### Adaptive Quorum Voting
+
```

Comment:
tbh just reference Polkadot docs

Reply:
because it does not contains our details of parameters

Reply:
edited